content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0.md

```diff
@@ -10,6 +10,17 @@ This release of Cumulocity IoT Streaming Analytics includes the Apama version 10
 EPL apps developers should also refer to [What's New In Apama 10.15.3](https://documentation.softwareag.com/pam/10.15.3/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-WhaNewInApa_10153_top.html)
 in the Apama documentation.
 
+### Restructured Streaming Analytics guide
+
+The Analytics Builder documentation, which was available separately until release 10.17, has been moved to the [Streaming Analytics guide](https://cumulocity.com/guides/10.18.0/streaming-analytics/overview-streaming-analytics/).
```

Comment:
@kailol-sag separately > better also use "as a separate webhelp" to be clearer
Also, isn´t it "which is available until release 10.17"?

Reply:
Done, thanks.